### PR TITLE
update known amendments voting status

### PR DESCRIPTION
**Pull request by oeggert**
Updating:
- [fixAMMOverflowOffer](https://xrpscan.com/amendment/fixAMMOverflowOffer)
- [fixDisallowIncomingV1](https://xrpscan.com/amendment/fixDisallowIncomingV1)
- [fixFillOrKill](https://xrpscan.com/amendment/fixFillOrKill)

DON'T MERGE UNTIL 04/11/24

### CODE REVIEW

File: resources/known-amendments.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ The following is a comprehensive list of all known [amendments](../docs/concepts
 
 | Name                              | Introduced | Status                        |
 |:----------------------------------|:-----------|:------------------------------|
-| [fixAMMOverflowOffer][]           | v2.1.1     | [Open for Voting: 2024-03-27](https://xrpl.org/blog/2024/rippled-2.1.1.html "BADGE_80d0e0") |
+| [fixAMMOverflowOffer][]           | v2.1.1     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.1.1.html "BADGE_GREEN") |
```
Comment (mDuo13) [inline resources/known-amendments.md]: Change the URL to an explorer link with the EnableAmendment pseudo-tx when it goes live. Same on lines 24-25 below for the other two.

**Review by mDuo13 — COMMENTED** [review]
In the future, probably best to make it a draft PR if it's a "don't merge until [date]" situation.

File: resources/known-amendments.md (hunk 1 above)
Comment (amarantha-k) [inline resources/known-amendments.md]:
```suggestion
| [fixAMMOverflowOffer][]           | v2.1.1     | [Enabled: 2024-04-11](https://livenet.xrpl.org/transactions/64144409D991726D108B89D79F9305438D61928A322EF1CD14DC3A5F24CE64BC "BADGE_GREEN") |
```

File: resources/known-amendments.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ The following is a comprehensive list of all known [amendments](../docs/concepts
...
 | [fixInnerObjTemplate][]           | v2.1.0     | [Open for Voting: 2024-02-20](https://xrpl.org/blog/2024/rippled-2.1.0.html "BADGE_80d0e0") |
```
Comment (amarantha-k) [inline resources/known-amendments.md]: This was enabled on Apr 08.
```suggestion
| [fixInnerObjTemplate][]           | v2.1.0     | [Enabled: 2024-04-08](https://livenet.xrpl.org/transactions/EC67D9DF8D06067A76E8F8F43BC036B5E0267568F8D92624A658AC01A8186235 "BADGE_GREEN") |
```

File: resources/known-amendments.md [hunk 3]
```diff
@@ -17,12 +17,12 @@ The following is a comprehensive list of all known [amendments](../docs/concepts
...
 | [DID][]                           | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
-| [fixDisallowIncomingV1][]         | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
-| [fixFillOrKill][]                 | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
+| [fixDisallowIncomingV1][]         | v2.0.0     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_GREEN") |
+| [fixFillOrKill][]                 | v2.0.0     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_GREEN") |
 | [XChainBridge][]                  | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
 | [AMM][]                           | v1.12.0    | [Enabled: 2024-03-22](https://livenet.xrpl.org/transactions/75F52BB86416717288999523063D54E24290EFEA2E99DF78E80A12BD1C8FAC99 "BADGE_GREEN") |
 | [XRPFees][]                       | v1.10.0    | [Open for Voting: 2023-03-14](https://xrpl.org/blog/2023/rippled-1.10.0.html "BADGE_80d0e0") |
```
Comment (amarantha-k) [inline resources/known-amendments.md]:
```suggestion
| [XRPFees][]                       | v1.10.0    | [Enabled: 2023-03-14](https://livenet.xrpl.org/transactions/4B6047F84B959B64FDD10E22D9E7CCC1EA0D228387462E8FF975B17F7C779021 "BADGE_GREEN") |
```

File: resources/known-amendments.md [hunk 4]
```diff
@@ -17,12 +17,12 @@ The following is a comprehensive list of all known [amendments](../docs/concepts
...
-| [fixAMMOverflowOffer][]           | v2.1.1     | [Open for Voting: 2024-03-27](https://xrpl.org/blog/2024/rippled-2.1.1.html "BADGE_80d0e0") |
+| [fixAMMOverflowOffer][]           | v2.1.1     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.1.1.html "BADGE_GREEN") |
 | [fixInnerObjTemplate][]           | v2.1.0     | [Open for Voting: 2024-02-20](https://xrpl.org/blog/2024/rippled-2.1.0.html "BADGE_80d0e0") |
 | [fixNFTokenReserve][]             | v2.1.0     | [Open for Voting: 2024-02-20](https://xrpl.org/blog/2024/rippled-2.1.0.html "BADGE_80d0e0") |
 | [DID][]                           | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
-| [fixDisallowIncomingV1][]         | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
-| [fixFillOrKill][]                 | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
+| [fixDisallowIncomingV1][]         | v2.0.0     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_GREEN") |
```
Comment (amarantha-k) [inline resources/known-amendments.md]: +1 to Rome's comment above - please update the URL for fixDisallowIncomingV1 and fixFillOrKill amendments once they are live.

**Review by amarantha-k — CHANGES_REQUESTED** [review]

**Review by amarantha-k — APPROVED** [review]
LGTM

**Pull request merged**